config/loadable_html/extension.rb

```diff
@@ -12,55 +12,52 @@ def process document, output
 
     if document.attr? 'stem'
 
-      loading_msg = '<div id="loading_msg" class="hidden" hidden><p>Loading&hellip; please wait.</p></div>'
-      loadable_class = 'class="loadable"'
+      logo_bug_white = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAACXBIWXMAAC4jAAAuIwF4pT92AAAAB3RJTUUH5AYdEwQiw14QHwAAAhpJREFUaN7tWO1xgzAMfc5lADpBGYFO0GxQOkHpBE03YANGyAiQCegGJBMkG5AN1D/KHSXGXxinufO74/gBSHrSk2wMRERERESEABHVpEbxH20PnRQaJ7Wj3YT0SHwQWMTR3MSsTB0JIS4AGs1rG4fcvGme7332gS5bu9BVXVtyaAB8ATgDOPL9PHh+sbSX6/xx5f0QYGMvHofbhROQuspHzJTUVfOpJIhhdc5CiLPCTgqgAPAxsvOkq4CwCDblkr8CyBRZU+HA1xHAjxDiIPGTs0wvQoh3X43b0TLoiWjHQd80+NzAN0R0onA4EVHpa9Gq6H7oZxG5c/A3RKymEE+V1nEcNtycB9nU4YxmAK73Z75nBo3/KWt4l13hGK2sAR37bcv++wlfpYmh3qK8ssmRcjAlX6nknUynb36nkgyRWvmtRfDZKOhyYmJtJT5OAzstf5tPBcbPhsroJkkYzvvNKHhV1TpJZlWoVAsprxnTW2yDCdQqMqrdTRrYLwxknsikaZrRrQPpwqLCXhaxrcJBPtFw2j8qTo73X9IpEjsbjZrISJMYPz/uBiROjr1TBJGPJLDSsA90MuqCycdgV/pnHbCQUTj5TIyuSkVi5iYwQQiMFpPxipw7Bl8jNAZESse9VDj5eBq/i8ln5ZmD7Sma9tznHlXoH1I+jjKaPX1WC3DYP6x8LGXkRT7rhTh8G5zcNYiIiIh4ePwCG/5IoFeZaBUAAAAASUVORK5CYII='
+      loading_msg = '<div id="loading_msg">Loading <img alt="" class="loading_spinner" decoding="sync" src="' + logo_bug_white + '" /></div>'
+      noJS_class = 'class="noJS"'
```

Comment:
This looks very likely to conflict with #1300 since they're changing the same lines in inconsistent ways.

Reply:
This is optimistically stacked PR on top of #1300. The first commit is the #1300.
Due to how git works, Tthere might be conflict when (and if #1300) is merged, but the rebase should be trivial.